backend/routes/contactCreator.ts

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable @typescript-eslint/no-misused-promises */
+import { type User } from '@prisma/client'
+import express from 'express'
+import prisma from '../utils/prismaClient'
+
+const contactCreatorRouter = express.Router()
+const db = prisma
```

Comment:
does `import db from '../utils/prismaClient'` not work?

Reply:
yeah this worked thanks!